You can pass an `Error` object to `captureException()` to have it captured as an event. It's also possible to pass non-`Error` objects and strings, but be aware that the resulting events in Sentry may be missing a stack trace.

```javascript
import * as Sentry from "@sentry/bun";

try {
  aFunctionThatMightFail();
} catch (e) {
  Sentry.captureException(e);
}
```
